example/sharding/index.test.js

```diff
@@ -1,4 +1,4 @@
-const indexfile = require("./index.js");
```

Comment:
Do we want this change in a `sharding` test? Rename the file instead? Or don't change this example? Doesn't really matter though 🤷 

Interesting that this had the `.js` extension in the require statement 🤔

Reply:
It's just redundant in a require statement so may as well make our examples clean without redundancies. Node will look for an index.js or index.mjs or index.cjs if you require a folder.

Reply:
Yeah I don't think we should ever have the file extension in there.

But an index is normally to group a folder so the folder can be imported while hiding how many files it has in it, not a random file like this for importing directly. Doesn't really relate to this PR though 🤷

Reply:
Yeah. Suppose it is a contrived example in that way.